pypesto/store/read_from_hdf5.py

```diff
@@ -331,3 +339,45 @@ def read_result(filename: str,
                            f'within {filename}.')
 
     return result
+    
+    
+def read_ensemble_prediction_from_h5(predictor: Callable[[Sequence], PredictionResult],
+                                     input_file: str,
+                                     base_path: str = None):
+    # ToDo read bounds
+    base = Path('')
+    if base_path is not None:
+        base = Path(base_path)
+
+    # open file
+    f = h5py.File(input_file, 'r')
```

Comment:
Rather make it into `With h5py.File(input_file, 'r') as f:` that way we ensure it is closed after reading.